src/main/java/com/minsta/m/global/error/handler/GlobalExceptionHandler.java

```diff
@@ -22,6 +29,37 @@ public ResponseEntity<ErrorMessage> globalExceptionHandler(HttpServletRequest re
         return new ResponseEntity<>(errorMessage, HttpStatus.valueOf(e.getErrorCode().getStatus()));
     }
 
+    @ExceptionHandler({MethodArgumentNotValidException.class, HttpMessageNotReadableException.class})
+    public ResponseEntity<ErrorMessage> validateException(MethodArgumentNotValidException e) {
+        log.warn("Validation Failed : {}", e.getMessage());
+        log.trace("Validation Failed Details : ", e);
+        return new ResponseEntity<>(new ErrorMessage(methodArgumentNotValidExceptionToJson(e), 400), HttpStatus.BAD_REQUEST);
+    }
+
+    @ExceptionHandler(RuntimeException.class)
+    public ResponseEntity<ErrorMessage> unExpectedException(RuntimeException e) {
+        log.error("UnExpectedException Occur : ", e);
+        return new ResponseEntity<>(new ErrorMessage("Internal Server Error", 500), HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+    @ExceptionHandler(NoHandlerFoundException.class)
+    public ResponseEntity<ErrorMessage> noHandlerFoundException(NoHandlerFoundException e) {
+        log.warn("Not Found Endpoint : {}", e.getMessage());
+        log.trace("Not Found Endpoint Details : ", e);
+        return new ResponseEntity<>(new ErrorMessage(HttpStatus.NOT_FOUND.getReasonPhrase(), 404), HttpStatus.FORBIDDEN);
```

Comment:
I think noHandler is forbidden status, because mean of not found status if for resource but no handle is not resource.
so i think forbidden status is appropriated